src/main/java/smartthings/konsumer/filterchain/MessageFilterChain.java

```diff
@@ -0,0 +1,57 @@
+package smartthings.konsumer.filterchain;
+
+import kafka.message.MessageAndMetadata;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import smartthings.konsumer.MessageProcessor;
+import smartthings.konsumer.circuitbreaker.CircuitBreaker;
+
+import java.util.*;
+
+public class MessageFilterChain {
```

Comment:
Maybe not necessary since this is more of an internal class from the looks of it.